apps/axiom-e2e/src/index.spec.ts

```
@@ -0,0 +1,59 @@
+import { Ensure, equals } from '@serenity-js/assertions'
+import { Task } from '@serenity-js/core'
+import { describe, it } from '@serenity-js/playwright-test'
+import { Navigate, Page } from '@serenity-js/web'
+
+export function startLoggedOut() {
+  return Task.where('#actor starts logged out', Navigate.to('/'), Ensure.that(
+    Page.current().title().describedAs('Login -cuHacking 2025'),
+    equals('Dashboard - cuHacking Portal'),
+  ))
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)